zcash_primitives/src/transaction/builder.rs

```diff
@@ -972,6 +975,7 @@ impl<'a, P: consensus::Parameters, U: sapling::builder::ProverProgress> Builder<
         #[cfg(zcash_unstable = "nu6" /* TODO nu7 */ )]
         let issue_bundle = unauthed_tx
             .issue_bundle
+            .map(|b| b.update_rho(first_nullifier(&orchard_bundle).unwrap()))
```

Comment:
what's the point of `Result<>` if we don't check it?

Reply:
Not so much indeed, Error scenario is extremely unlikely here, so we can simplify the flow to just panic inside the function in few extremely rare situation

Reply:
ok, let's remove `result<>`